testsuite/integration-arquillian/tests/base/testsuites/database-suite

```diff
@@ -0,0 +1,26 @@
+AccountRestServiceTest
```

Comment:
How do we prevent this list from becoming obsolete, e.g. when a test is renamed / deleted etc?

Reply:
This is an initial approach for the testsuite definition, and I'm not 100% happy with it, so will try to come up with something better and more maintainable, but would like to get this in for now

Reply:
That works for me, thanks. Please consider this conversation resolved once there is a issue for this follow-up